### DISCUSSION TITLE
IDE 설정 통일

### DISCUSSION THREAD

**Discussion by Choi-JJunho** [woowacourse-teams/2023-pium#233] [개발 논의]
# 주제
IDE 설정을 통일해야할 것 같습니다

이유는 다음과 같습니다.

- 라인정렬 기준이 달라짐
- import 정렬 및 * 사용 유무가 다름

위와 같은 상황으로 인해 각자의 PR이 작성되었을 때 불필요한 File changes가 발생합니다

## 제안
이에 인텔리제이의 설정을 통일하고 해당 내용을 문서화하여 컨벤션으로 지정하는건 어떨까요?

**Comment by yeonkkk** [comment]
동의합니다. 한 번 맞추고 가면 좋을 것 같아요~
오늘 회의 해봐요!

**Comment by Choi-JJunho** [comment]
- Wiki에 [인텔리제이 IDE 설정](https://github.com/woowacourse-teams/2023-pium/wiki/%5BBE%5D-%EC%9D%B8%ED%85%94%EB%A6%AC%EC%A0%9C%EC%9D%B4-%ED%99%98%EA%B2%BD%EC%84%A4%EC%A0%95) 추가했습니다.
